nbextensions/appCell2/widgets/appCellWidget-fsm.js

```diff
@@ -132,12 +132,69 @@ define([], () => {
             },
         };
 
+    const STATE = {
+        NEW: {
+            mode: 'new',
+        },
+        INTERNAL_ERROR: {
+            mode: 'internal-error',
+        },
+        EDITING_INCOMPLETE: {
+            mode: 'editing',
+            params: 'incomplete',
+        },
+        EDITING_COMPLETE: {
+            mode: 'editing',
+            params: 'complete',
+            code: 'built',
+        },
+        EDITING_BATCH_INCOMPLETE: {
+            mode: 'editing-batch',
+            params: 'incomplete',
+        },
+        EDITING_BATCH_COMPLETE: {
+            mode: 'editing-batch',
+            params: 'complete',
+            code: 'built',
+        },
+        EXECUTE_REQUESTED: {
+            mode: 'execute-requested',
+        },
+        PROCESSING_LAUNCHED: {
+            mode: 'processing',
+            stage: 'launched',
+        },
+        PROCESSING_QUEUED: {
+            mode: 'processing',
+            stage: 'queued',
+        },
+        PROCESSING_RUNNING: {
+            mode: 'processing',
+            stage: 'running',
+        },
+        COMPLETED: {
+            mode: 'completed',
+        },
```

Comment:
I changed this from `mode: 'success'` to `completed` so that it matches the job status